tests/cmd/generate/oci_image/test_module_init.py

```diff
@@ -0,0 +1,171 @@
+import json
+from pathlib import Path
+from typing import Any
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from mobster.cmd.generate.oci_image import GenerateOciImageCommand
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    [
+        "syft_boms",
+        "hermeto_bom",
+        "image_pullspec",
+        "image_digest",
+        "parsed_dockerfile_path",
+        "dockerfile_target_stage",
+        "additional_base_images",
+        "contextualize",
+        "expected_sbom_path",
+    ],
+    [
+        (
+            [Path("tests/sbom/test_merge_data/spdx/syft-sboms/pip-e2e-test.bom.json")],
+            Path("tests/sbom/test_merge_data/spdx/cachi2.bom.json"),
+            "quay.io/foobar/examplecontainer:v10",
+            "sha256:1",
```

Comment:
I am surprised this passes a validation of spdx or cyclonedx libraries as a valid checksum. Do you validate the content when loading it to library representations?

Reply:
So far I have only created a small validation in `cli.py` (which this test skips). The validation just checks that the hash starts with `sha256:`, but I can extend it to use a regex like `r"sha256:[0-9a-f]{32}"`.

Reply:
Added validation of the `execute()` method with logging the returned messages as warnings.